src/control/cmd/dmg/storage_test.go

```diff
@@ -50,34 +55,32 @@ func TestStorageCommands(t *testing.T) {
 			"Format with reformat",
 			"storage format --reformat",
 			strings.Join([]string{
+				"ConnectClients",
+				printRequest(t, &control.SystemQueryReq{}),
 				"ConnectClients",
 				printRequest(t, &control.StorageFormatReq{Reformat: true}),
 			}, " "),
 			nil,
 		},
 		{
-			"Reformat system without ranks filter",
-			"storage format --system",
+			"Format with invalid ranks filter",
+			"storage format --ranks 1-3",
 			strings.Join([]string{
 				"ConnectClients",
-				printRequest(t, &control.SystemResetFormatReq{Ranks: []system.Rank{}}),
-				"ConnectClients",
-				printRequest(t, storageFormatReq),
+				printRequest(t, &control.StorageFormatReq{}),
 			}, " "),
-			nil,
+			errors.New("--ranks parameter invalid"),
 		},
 		{
-			"Reformat system with ranks filter",
-			"storage format --system --ranks 0,1,2,3,4", // TODO DAOS-4454: enable [0-4] syntax
+			"Non-system reformat with invalid ranks filter",
+			"storage format --reformat --ranks 0-4",
```

Comment:
Are we still testing the system format path somewhere?

Reply:
yes , I've added tests for checkSystemReformat()